libs/fluid-elements/tab-group/src/lib/tab/tab.scss

```diff
@@ -56,9 +56,13 @@
     left: 0;
     content: '';
   }
+
+  &:not(.fluid-tabbed):focus {
+    outline: none;
+  }
 }
 
-.fluid-state--active {
+.fluid-tab-state--active {
```

Comment:
This was intentionally designed to not contain the element, but be a more generic class name that holds state representation. But there is no guildeline for this one yet, so it's fine.

Reply:
I'll gladly change it back. When i was reading over it I thought it is more descriptive of what it styles and forgot we talked about these classes to be more generic.